ui/admin/view.php

```diff
@@ -182,24 +180,18 @@
 				}//end if
 
 				if ( 0 < count( $fields ) ) {
+					if ( $more ) {
+						$title = __( 'More Fields', 'pods' );
+					} else {
+						$title = __( 'Fields', 'pods' );
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $title

<sub>hash: 9e665f2698124fd693173ea75e6e5c2f</sub>